kyuubi-server/src/main/scala/org/apache/kyuubi/engine/ProcBuilder.scala

```diff
@@ -56,13 +57,14 @@ trait ProcBuilder {
     }
   }
 
+  // TODO use SPARK_SCALA_VERSION instead of server's runtime scala version
+  protected val jarName: String = s"${module}_$serverRuntimeScalaVersion-$KYUUBI_VERSION.jar"
```

Comment:
pullout the `engineScalaBinaryVersion` should be sufficient.

Reply:
Good Suggestion. Applied.